index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
             </div>
         </section>
     </div>
-   
+
+    <!-- Work Section -->
+    <section>
+        <div class="work-container">
+            <div class="work-container-background">
+                <div class="work-image-container">
+                    <img src="./Assets/mobile_Snapshoot_1.png" alt="">
+                </div>
+    
+                <div class="work-text-container">
+                    <h1>Tonic</h1>
+                    <div class="work-complementary-text">
+                        <ul class="work-complementary-list">
+                            <li class="canopy">CANOPY</li>
+                            <li class="canopy-complement"><img src="./Assets/Counter.png" alt="">Back End Dev</li>
+                            <li class="canopy-complement"><img src="./Assets/Counter.png" alt="">2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div>
+                        <ul class="buttons-work-container">
+                            <li class="btn-work-style">html</li>
+                            <li class="btn-work-style">css</li>
+                            <li class="btn-work-style">Javascript</li>
+                        </ul>
+                    </div>
+                    <button class="button-work-project">SEE PROJECT</button>
+                </div>
```

Comment:
### See Project Button
![Capture3](https://user-images.githubusercontent.com/93039359/198689648-81cb722d-f6a8-40d2-86f9-e94341e30b12.PNG)

- [ ] Kindly add these effects on your button through css to make your design look similar to the figma template to make your design professional 👍